src/Bloc-Animation/BlTransformAnimation.class.st

```diff
@@ -6,10 +6,13 @@ Transform animation can be absolute or relative.
  - #relative builds on existing element transformation
  - #absolute defines new element independent transformation
 
+BlTransformAnimation use BlElementAbsoluteTransformation to manage the transformation matrix, even if each transformation use BlElementLocalTransformation for its defintion.  This transformation works by updating its transformation matrix, from IdentityMatrix, to target matrix, through interpolation computed at each animation step (fromMatrix interpolate: aNumber to: toMatrix)
+
+This transformation doesn't use an origin point. Instead, It'll apply a translation, so that your shape still look like it's in the element bound. This translation is added to your transformation definition (boundingRectangle:)
```

Comment:
Concretely, this is the comment to add, right? if this is the case, I would re-do the commit only with it. Because there is a not of noise due to Tonel version change. I don't know why this happened, since repo explicits the version to use.

Reply:
I mean, Iceberg should had taken the repo version automatically at the comment of pressing the Commit button in your IDE.